core/templates/dev/head/pages/review-test-page/review-test-page.directive.ts

```diff
@@ -68,10 +78,30 @@ oppia.directive('reviewTestPage', ['UrlInterpolationService', function(
                 result.data.skill_descriptions[skillId]);
             }
             var questionPlayerConfig = {
+              resultActionButtons: [
+                {
+                  type: 'BOOST_SCORE',
+                  text: 'Boost My Score'
```

Comment:
Do these need to be translated? As these are learner facing. If so, you should add these as I18N strings.

Reply:
I had a question about this comment - This string will be rendered on the HTML page. Considering that we have been writing regular English text on HTML as well, won't this get translated the same way as any other strings in the HTML file?

Reply:
Actually no. So if you see, we have very specific strings that get translated. You'll need to use the translate="I18N_..." attribute in the tag to translate it. The files which store the translated versions of these I18N strings are regularly updated (you'll see PRs from translatewiki titled "routine update of translations"). This is how we manage to do translations of text. Now for views that are learner facing, this is crucial (not so much for creator views). So you would want to add I18N strings for all headings, text, etc on the players. Let me know if you need some more explanation. But you could just find an I18n string, and trace it back to figure out what to add and where. I think you'll need to edit a couple of JSON files to do this (the english version, and qqq.json I think).

Reply:
Oh I see - so what you are saying is that even in the HTML page we have to add the I18N attributes for every piece of text right?

Reply:
Yeah. You could look at the library page, or the learner dashbaord for reference.

Reply:
But do we need to add every I18N string for every text in this case? I don't think we are doing this for HTML files. If that's the case, we might need to do the cleanup for many files instead of only this one text.

Reply:
It needs to happen for anything that's learner-facing. Creator-facing views aren't internationalized.

Reply:
Ok I see. I think it is an issue for question player in general then. I will create an issue and resolve it in a separate PR.

Reply:
https://github.com/oppia/oppia/issues/7084 issue created